src/msg/async/dpdk/dpdk_rte.cc

```diff
@@ -159,7 +163,7 @@ namespace dpdk {
     std::unique_lock<std::mutex> l(lock);
     while (!done)
       cond.wait(l);
-    return 0;
+    return initialized ? 0:-EIO;
```

Comment:
nit, not quite related to this change, but the code could be refactored so it reads:
```c++
std::unique_lock locker(lock);
cond.wait(locker, [&] { return done; });
return initialized ? 0 : -EIO;
```

Reply:
please add spaces around `:`.

Reply:
Ok

Reply:
> nit, not quite related to this change, but the code could be refactored so it reads:
> 
> ```c++
> std::unique_lock locker(lock);
> cond.wait(locker, [&] { return done; });
> return initialized ? 0 : -EIO;
> ```

Ok